src/Modal.js

```diff
@@ -1,10 +1,11 @@
-import propTypes from '@dhis2/prop-types'
-import React from 'react'
 import { createPortal } from 'react-dom'
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
 
-import { sizePropType } from './common-prop-types.js'
-import { ScreenCover } from './ScreenCover.js'
 import { ModalCard } from './Modal/ModalCard.js'
+import { ScreenCover } from './ScreenCover.js'
+import { insideAlignment } from './common-prop-types'
+import { sizePropType } from './common-prop-types.js'
```

Comment:
on one line

Reply:
Also perhaps keep using the naming convention that's already present, i.e. use the `PropType` suffix